open-craft/opencraft
# Upgrade PyYAML and remove some unused dependencies

This is a follow-up of https://github.com/open-craft/opencraft/pull/436. It upgrades some packages with vulnerabilities (according to github):
- it upgrades PyYAML, which wasn't possible before due to version conflicts, but now it is
- it removes ansible and its dependencies, since it wasn't initially in requirements.txt and it was added by mistake
- it does some minor upgrades to stay up to date (`pip-upgrade -U`)


Testing:
- similar to the cited PR
- CI tests should pass
- deploy to stage and test
- deploy to production
